### ISSUE TITLE
Better Questing Spams on start of new world

### ISSUE THREAD

**Issue by ChromeMystic** [scalda/FoolCraft#638]
## SUGGESTION/HELP
- [ ] I would like to propose an additional mod, config change, custom recipe, etc.
- [ ] I am requesting help with installing or running the pack

## Issue Description
When Starting a new World Betterquesting Spams the The Screen with all the quests. "Quest Complete - New Quest" to be specific with chaning items.

## Reproduce Steps
Start a new World

**Comment by scalda** [comment]
I am aware we are working on it still

**Comment by Tegwen** [comment]
I couldn't figure out what was going on as I was mining stone and the cobble would go into my inventory and then vanish.  I finally figured out that the last quest I had completed was Cobble Gobbler which consumes a stack of cobblestone to complete.  So the quest kept "completing" and eating all the cobblestone.  I found that I quickly tried to complete another quest and after a short while that quest was completing instead and I could finally gather cobblestone.
